This is the jupyter notebook which will hold our neural networks and be used to create graphs and stuff

In [2]:
import numpy as np
X_test = np.load("project/X_test.npy")
y_test = np.load("project/y_test.npy")
person_train_valid = np.load("project/person_train_valid.npy")
X_train_valid = np.load("project/X_train_valid.npy")
y_train_valid = np.load("project/y_train_valid.npy")
person_test = np.load("project/person_test.npy")

sets = (X_test, y_test, person_train_valid, X_train_valid, y_train_valid, person_test)
labels = ["X_test",
          "y_test",
          "person_train_valid",
          "X_train_valid",
          "y_train_valid",
          "person_test"]
for i, k in enumerate(sets):
    print('{}: {} '.format(labels[i], k.shape))

X_test: (443, 22, 1000) 
y_test: (443,) 
person_train_valid: (2115, 1) 
X_train_valid: (2115, 22, 1000) 
y_train_valid: (2115,) 
person_test: (443, 1) 


There are 2115 trials and 1000 timestamps. There are 22 electrodes. There are 443 test trials. There are persons 0-8 (inclusive) and person_train_valid[trial] gives the person that was used in the specific trial. 